STYLE_GUIDE.md

```diff
@@ -118,6 +128,22 @@ A procedure is a series of numbered steps that a user follows to complete a spec
 - Parallel language constructions
 - Consistent formatting
 
+Replace pointer-specific language with device-agnostic language to accommodate readers with disabilities and users of various input methods and devices, including the pointer, keyboard, and touch screens.
+
+For example, instead of the term *click*, which is pointer-specific, use *choose*, which is more generic and device-agnostic. However, when the generic language makes it difficult to understand the instructions (for example, in the case of opening a context menu), you can include pointer-specific hints. Use your judgment. If you have a question, ask your editor.
+
+Use the following language to describe UI interactions:
```

Comment:
Could we also add a list of our Dashboards components and how to refer to them? For example, do we call the three-line menu "Top menu", "Menu", "Hamburger menu" or something else?